ark/segmentation/marker_quantification_test.py

```diff
@@ -179,14 +202,22 @@ def test_generate_expression_matrix_multiple_compartments():
     nuc_masks *= 2
 
     unequal_masks = np.concatenate((cell_masks, nuc_masks), axis=-1)
-    coords = [["Point0", "Point1"], range(40), range(40), ['whole_cell', 'nuclear']]
-    dims = ['fovs', 'rows', 'cols', 'compartments']
-    segmentation_masks_unequal = xr.DataArray(unequal_masks, coords=coords, dims=dims)
 
-    channel_data = xr.DataArray(channel_datas,
-                                coords=[["Point0", "Point1"], range(40), range(40),
-                                        ["chan0", "chan1", "chan2", "chan3", "chan4"]],
-                                dims=["fovs", "rows", "cols", "channels"])
+    segmentation_masks_unequal = test_utils.make_labels_xarray(
+        label_data=unequal_masks,
+        fov_ids=fovs,
+        row_size=40,
+        col_size=40,
```

Comment:
Same thing here, we should be able to get rid of almost all of these arguments since they're defaults that are constant throughout